templates/ecs.yaml

```diff
@@ -0,0 +1,317 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  KeyName:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: Name of an existing EC2 KeyPair to enable SSH access to the ECS instances.
+  VpcId:
+    Type: AWS::EC2::VPC::Id
+    Description: Select a VPC that allows instances access to the Internet.
+  BastionHostSgId:
+    Type: String
+    Description: The ID of the security group for bastion hosts
+  PublicSubnetId:
+    Type: List<AWS::EC2::Subnet::Id>
+    Description: Public subnet IDs from the VPC for ALB placement. Minimum of two.
+  PrivateSubnetId:
+    Type: List<AWS::EC2::Subnet::Id>
+    Description: Private subnet IDs from the VPC for ECS node placement. Minimum of two.
+  DesiredCapacity:
+    Type: Number
+    Default: '1'
+    Description: Number of instances to launch in your ECS cluster.
+  MaxSize:
+    Type: Number
+    Default: '1'
+    Description: Maximum number of instances that can be launched in your ECS cluster.
+  InstanceType:
+    Description: EC2 instance type
+    Type: String
+    Default: t3a.micro
+    AllowedValues: [t3.micro, t3.small, t3.medium, t3.large, t3a.micro, t3a.small, t3a.medium, t3a.large]
+    ConstraintDescription: Please choose a valid instance type.
+  DockerImageTag:
+    Type: String
+    Description: The Docker image tae to run
+Mappings:
+  AWSRegionToAMI:
+    us-east-1:
+      AMIID: ami-0669eafef622afea1
```

Comment:
These are the current ECS-optimized Amazon Linux 2 AMIs for the two regions we've used this repo in.

Reply:
Our experience is that stock AMIs can disappear from AWS because AWS will retire AMIs regularly.  I'm guessing the probably do the same thing with  ECS-optimized AMIs as well.  My concern is that this specific AMI will no longer be available over time.  it should probably reference the latest ECS AMI.  Example of that is here https://aws.amazon.com/blogs/compute/query-for-the-latest-amazon-linux-ami-ids-using-aws-systems-manager-parameter-store/